client/src/Components/RatingsAndReviews/RatingsAndReviewsIndex.jsx

```diff
@@ -105,7 +105,7 @@ var RatingsAndReviewsIndex = (props) => {
 
   // console.log(meta, 'reviiiews')
   return (
-    <div>
+    <div id="Reviews">
```

Comment:
@dondo5252 I added `id="Reviews"` to the wrapper of your RatingsAndReviewsIndex.jsx. Pinging to ask if you're fine with that. It's for the purposes of auto-scrolling the page to the RatingsAndReviews section when the user clicks on "Read [#] Reviews" in the Product Overview.

Reply:
@Panlord Yes of course!